docs/pages/kubernetes-access/helm/guides/aws.mdx

```diff
@@ -167,13 +191,7 @@ You'll need to replace these values in the policy example below:
 }
 ```
 
-## Step 4/7. Install and configure cert-manager
-
-Reference: [`cert-manager` docs](https://cert-manager.io/docs/)
-
-In this example, we are using multiple pods to create a High Availability Teleport cluster. As such, we will be using
-`cert-manager` to centrally provision TLS certificates using Let's Encrypt. These certificates will be mounted into each
-Teleport pod, and automatically renewed and kept up to date by `cert-manager`.
+#### Installing cert-manager
```

Comment:
Is this supposed to be a subsection of the "Route53 IAM policy" section? Would it be more appropriate as an H3 section?